cli/flox/doc/flox-push.md

```diff
@@ -7,45 +7,57 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-push -
+flox-push - send environment to FloxHub
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] push [ `<options>` ] [ \--force ] [ ( -m | \--main ) ]
+```
+flox [ <general-options> ] push
+     [-d=<path>]
+     [-o=<owner>]
+     [-f]
+```
 
 # DESCRIPTION
 
-(`git`) Push metadata to the environment's `floxmeta` repository.
-With this mechanism environments can be pushed and pulled between machines
-and within teams just as you would any project managed with `git`.
+Move a path environment to FloxHub or push local changes to a managed
+environment to FloxHub.
 
-With the `(-m|\--main)` argument `flox (push|pull)` will operate on the
-"floxmain" branch, pushing user metadata to the upstream repository.
-Cannot be used in conjunction with the `-e|\--environment` flag.
+After pushing, the remote environment can be referred to as `<owner/name>`.
 
-With the `--force` argument flox will forceably overwrite either the
-upstream or local copy of the environment based on having invoked
-`push` or `pull`, respectively.
+A path environment contains a manifest file and lock file stored locally and
+possibly committed to version control.
```

Comment:
```suggestion
A path environment contains a manifest file and lock file stored locally
and possibly committed to version control.
```

Reply:
I'm not sure what your reasoning is for this? Are you thinking it's sembr, and if so can you point me at a rule?

Reply:
Yeah sembr, rule 6 I guess

Reply:
`stored locally and possibly committed to version control` is a phrase that's part of the clause that makes up the entire sentence, so rule 6 doesn't apply. And rule 6 is optional